polaris-migrator/src/migrations/replace-sass-z-index/replace-sass-z-index.ts

```diff
@@ -0,0 +1,119 @@
+import type {API, FileInfo, Options} from 'jscodeshift';
+import postcss, {Plugin} from 'postcss';
+import valueParser, {FunctionNode, Node} from 'postcss-value-parser';
+
+import {POLARIS_MIGRATOR_COMMENT} from '../../constants';
+
+interface PluginOptions extends Options {
```

Comment:
`utils` now exposes a `NamespaceOptions` type we can extend here e.g.
`inferface PluginOptions extends Options, NamespaceOptions {}`